packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -26,7 +30,7 @@
     display: inline-block;
     font-size: var(--terra-time-input-font-size, 1.14286rem);
     line-height: var(--terra-time-input-line-height, 1.375);
-    margin: 0;
+    margin: 0 0.5rem 0.5rem 0;
```

Comment:
I am a bit sceptical about this change, not sure if this might impact the existing consumers of time input.
@neilpfeiffer, your thoughts on this?

Reply:
It is done to accommodate the design requirement mentioned [here](https://github.com/cerner/terra-framework/issues/706#issuecomment-518269645). I will want the bottom margin also to be 7px to accommodate the meridiem buttons under the time-input component when the viewport squishes.